docs/src/components/pricing/PricingTable.tsx

```diff
@@ -339,19 +339,25 @@ const rowHeaders: Record<string, React.ReactNode> = {
   'MUI Base': (
     <ColumnHead
       label="MUI Base"
-      tooltip="Unstyled components and react hooks available in @mui/base."
+      tooltip="A library of 'unstyled' components and low-level hooks, available @mui/base."
```

Comment:
@samuelsycamore I've removed the "in" from _"available in @mui/base"_ (and others) given "@" reads as "at". Is it correct? 😅

Reply:
hmm that's a good question. My gut wants to say that it's "the `@mui/base` package", therefore you would say it's "in the package," or "in `@mui/base`. I guess in the context of packages, I think of the `@` more as a signifier of what it is than something to be read in the name. But I could be wrong about that!

Reply:
that makes sense to me as well! but as we don't use the `@` when actually executing the install command in the terminal, maybe it's reasonable to read `@` as _"at"_?

Reply:
I would expect https://mui.com/base/getting-started/overview/'s description.

```suggestion
      tooltip="MUI Base is a library of headless ("unstyled") React UI components and low-level hooks, available in @mui/base."
```

Reply:
It could work too! I just copied the one from the READ.ME file.

Reply:
+1 to find a common description then, also with the package.json's description 😁

https://github.com/mui/material-ui/blob/4ab3a04c86aa5b7f4dfcc06fe2693cb9bd260c36/packages/mui-base/package.json#L6